neural_compressor/torch/algorithms/weight_only/modules.py

```diff
@@ -300,7 +302,253 @@ def unpack_tensor_with_torch(self, packed_tensor):
                 accelerator.synchronize()
         return unpacked_tensor
 
-    def pack_tensor_with_numpy(self, raw_tensor):
+    @staticmethod
+    @numba.jit(nopython=True, parallel=True)
+    def pack_array_with_numba_b4_c32(
```

Comment:
Please add UTs to verify the correctness of all pack functions.

Reply:
added https://github.com/intel/neural-compressor/pull/1883/commits/0ea77fd1ffdbf184abe7521a35f753079fb7bb54